roles/kubernetes/preinstall/tasks/0040-verify-settings.yml

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
add check about dual stack variable that 's deprecated as well as any other

Reply:
There is no check for deprecated in this test, we control all tests, does this check really make sense?